src/main.rs

```diff
@@ -176,6 +186,36 @@ async fn main() -> anyhow::Result<()> {
                 .required(false)
             )
         )
+        .subcommand(Command::new("build-ui")
+            .about("Build the web UI for an Uqbar process")
+            .arg(Arg::new("package-dir")
+                .action(ArgAction::Set)
+                .help("The package directory to build")
+                .default_value(&current_dir)
+            )
+            .arg(Arg::new("quiet")
+                .action(ArgAction::SetTrue)
+                .short('q')
+                .long("quiet")
+                .help("If set, do not print `cargo` stdout/stderr")
+                .required(false)
+            )
+        )
+        .subcommand(Command::new("build-with-ui")
+            .about("Build an Uqbar process and its web UI")
+            .arg(Arg::new("package-dir")
+                .action(ArgAction::Set)
+                .help("The package directory to build")
+                .default_value(&current_dir)
+            )
+            .arg(Arg::new("quiet")
+                .action(ArgAction::SetTrue)
+                .short('q')
+                .long("quiet")
+                .help("If set, do not print `cargo` stdout/stderr")
+                .required(false)
+            )
+        )
```

Comment:
As discussed, consider moving these instead to flags. E.g. `--ui` and `--ui-only` flags for the `build` subcommand.